reference-architecture/gce-cli/ansible/playbooks/roles/openshift-ansible-installer/tasks/main.yaml

```diff
@@ -0,0 +1,6 @@
+---
+- name: download openshift ansible installer from github
+  git:
+    repo: https://github.com/openshift/openshift-ansible.git
+    dest: '{{ playbook_dir }}/../../../../../openshift-ansible'
```

Comment:
I was looking into this, but I'm blocked by static playbook includes in ansible [1]. Tasks can be included dynamically, but playbooks not. So if I determine dynamically which playbook to include in openshift-setup.yaml [2] (based on check if installer is available as rpm), inclusion fails :-( You can see my non-working commit here [3]. Do you have any ideas how this could be solved?

[1] http://docs.ansible.com/ansible/playbooks_roles.html#dynamic-versus-static-includes
[2] https://github.com/openshift/openshift-ansible-contrib/blob/master/reference-architecture/gce-cli/ansible/playbooks/openshift-setup.yaml#L2
[3] https://github.com/pschiffe/openshift-ansible-contrib/commit/0229ae77c6e9aca2eb95a073266071147d34ae4a